shell/platform/fuchsia/dart_runner/BUILD.gn

```diff
@@ -77,20 +80,12 @@ runner("dart_jit_runner_bin") {
   if (flutter_runtime_mode == "profile") {
     extra_defines += [ "FLUTTER_PROFILE" ]
   }
-  extra_deps = [
```

Comment:
I get duplicate symbols with the stuff in "runtime:libdart" unless I delete these; same for below